b1ddi/data_source_ipam_next_available_ip.go

```diff
@@ -0,0 +1,253 @@
+package b1ddi
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	b1ddiclient "github.com/infobloxopen/b1ddi-go-client/client"
+	"github.com/infobloxopen/b1ddi-go-client/ipamsvc/address"
+	"strings"
+)
+
+func dataSourceIpamsvcNaIP() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIpamsvcNaIPRead,
+		Schema: map[string]*schema.Schema{
+
+			// The address in form "a.b.c.d".
+			// Required: true
```

Comment:
Comment says required while the code has `Optional: true`

Reply:
Updated